File: RS205kutuWorkspace/ArrayToStringComma.js
/*
モジュールArrayToStringComma 作成者: 佐々木 修正日時: 12/24
入力が配列ならその配列の各要素を文字列としてカンマで連結して返す
配列であるかの検査がojjectクラスであるかどうかまでしかできないので不安
*/

function ArrayToStringComma(array){
    // 返却する文字列
    let output = null; 
    // 配列であれば処理を行う
    if(array instanceof "object"){
        for(let i = 0; i < array.length; i++){
            output += array[i];
            // 配列の最後の要素にはカンマをつけないよう処理
            if(i < array.length - 1){
                output += ", ";
            }
        }
    }
    return output;
}

File: RS205kutuWorkspace/ArrayToStringComma.java
/*
モジュールArrayToStringComma 作成者: 佐々木 修正日時: 1/17
入力配列の各要素を文字列としてカンマで連結して返す
ChangeToCommaPatternメソッド呼び出し元モジュールで入力引数をString型の配列に変換する必要がある
*/

public class ArrayToStringComma{
    public String ChangeToCommaPattern(String[] array){
        String output = null;
        for(int i = 0; i < array.length; i++){
            output += array[i];
            if(i < array.length - 1){
                output += ", ";
            }
        }
        return output;
    }
}